history/kadai-simplehistory-rest-spring/src/test/java/io/kadai/simplehistory/rest/TaskHistoryEventControllerIntTest.java

```diff
@@ -54,11 +53,11 @@ class TaskHistoryEventControllerIntTest {
 
   private static final ParameterizedTypeReference<TaskHistoryEventPagedRepresentationModel>
       TASK_HISTORY_EVENT_PAGED_REPRESENTATION_MODEL_TYPE =
-          new ParameterizedTypeReference<TaskHistoryEventPagedRepresentationModel>() {};
+          new ParameterizedTypeReference<>() {};
 
   private static final ParameterizedTypeReference<TaskHistoryEventRepresentationModel>
       TASK_HISTORY_EVENT_REPRESENTATION_MODEL_TYPE =
-          new ParameterizedTypeReference<TaskHistoryEventRepresentationModel>() {};
```

Comment:
Why did you delete the type?

Reply:
Because the compiler infers it automatically: https://docs.oracle.com/javase/7/docs/technotes/guides/language/type-inference-generic-instance-creation.html